src/PDPVerifier.sol

```diff
@@ -274,7 +274,7 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
         emit ProofSetOwnerChanged(setId, oldOwner, msg.sender);
         address listenerAddr = proofSetListener[setId];
         if (listenerAddr != address(0)) {
-            try PDPListener(listenerAddr).ownerChanged(setId, oldOwner, msg.sender) {} catch {}
+            PDPListener(listenerAddr).ownerChanged(setId, oldOwner, msg.sender, "");
```

Comment:
Apologies I should have explained this better: please update the interface to claimProofSetOwnership to accept the extraBytes data.  The way this pattern works is that the caller can provide arbitrary data to be threaded through the listener call.  You'll need to add a `bytes` param to `claimProofSetOwnership` and then pass it in here instead of this empty string.